src/decideid_assets/package.json

```diff
@@ -0,0 +1,108 @@
+{
+  "name": "decideid_assets",
+  "version": "0.1.0",
+  "description": "Modclub",
+  "type": "module",
+  "keywords": [
+    "Internet Computer",
+    "Motoko",
+    "JavaScript",
+    "Canister"
+  ],
+  "scripts": {
+    "build": "node ../../scripts/build/gen_files_by_env.cjs && webpack",
+    "start": "export DEV_ENV=${DEV_ENV:-qa} &&  node ../../scripts/build/gen_files_by_env.cjs && webpack serve --mode development --config webpack.config.cjs"
+  },
+  "devDependencies": {
+    "@dfinity/agent": "^0.19.3",
+    "@dfinity/candid": "^0.19.3",
+    "@dfinity/principal": "^0.19.3",
+    "@types/react": "^17.0.27",
+    "assert": "2.0.0",
+    "autoprefixer": "^10.4.17",
+    "buffer": "6.0.3",
+    "copy-webpack-plugin": "^11.0.0",
+    "css-loader": "^6.9.1",
+    "esbuild-loader": "^3.0.1",
+    "events": "3.3.0",
+    "file-loader": "^6.1.1",
+    "html-webpack-plugin": "^5.5.0",
+    "postcss": "^8.4.33",
+    "postcss-loader": "^8.0.0",
+    "prettier": "^2.8.7",
+    "prettier-plugin-motoko": "^0.4.1",
+    "pretty-quick": "^3.1.3",
+    "process": "0.11.10",
+    "resolve-url-loader": "^5.0.0",
+    "sass": "^1.60.0",
+    "sass-loader": "^13.2.2",
+    "stream-browserify": "3.0.0",
+    "style-loader": "^3.3.4",
+    "terser-webpack-plugin": "^5.3.7",
+    "ts-loader": "^9.4.2",
+    "typescript": "^4.3.5",
+    "usergeek-ic-js": "1.0.5",
+    "util": "0.12.3",
+    "webpack": "^5.76.3",
+    "webpack-cli": "^5.0.1",
+    "webpack-dev-server": "^4.13.1"
+  },
+  "browserslist": [
+    "last 2 chrome version",
+    "last 2 firefox version",
+    "last 2 safari version",
+    "last 2 edge version"
+  ],
+  "dependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.14.5",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/plugin-transform-runtime": "^7.14.5",
+    "@babel/preset-env": "^7.14.8",
+    "@babel/preset-react": "^7.14.5",
+    "@connect2ic/core": "^0.2.0-beta.23",
+    "@connect2ic/react": "^0.2.0-beta.23",
+    "@connect2icmodclub/core": "0.0.4",
+    "@connect2icmodclub/react": "^0.0.1",
+    "@dfinity/auth-client": "^0.19.1",
+    "@dfinity/authentication": "^0.14.2",
+    "@dfinity/identity": "^0.19.1",
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^16.18.75",
+    "@types/react": "^18.2.48",
+    "@types/react-dom": "^18.2.18",
+    "classnames": "^2.3.2",
+    "date-fns": "^2.25.0",
+    "face-api.js": "^0.22.2",
+    "final-form": "^4.20.4",
+    "htmlparser2": "^9.0.0",
+    "ic-stoic-identity": "^2.0.0",
+    "mic-recorder-to-mp3": "^2.2.2",
+    "moment": "^2.29.4",
+    "postcss-preset-env": "^9.3.0",
+    "react": "^18.2.0",
+    "react-accessible-accordion": "^4.0.0",
+    "react-bulma-components": "^4.0.7",
+    "react-burger-menu": "^3.0.6",
+    "react-datepicker": "^4.8.0",
+    "react-dom": "^18.2.0",
+    "react-dropzone": "^11.4.2",
```

Comment:
do we really will use dependencies like this?

Reply:
let me clean up more unused.